app/scripts/migrations/105.ts

```diff
@@ -97,6 +97,7 @@ function createInternalAccountsForAccountsController(
       metadata: {
         name: identity.name,
         lastSelected: identity.lastSelected ?? undefined,
+        importTime: 0,
```

Comment:
Hmm, why was this change required? I'm guessing this was a type error, but would this impact other accounts in state as well? e.g. do we need a migration to add `importTime` to existing account metadata?

Reply:
Yea, its a type error. A default importTime was added on migration 119

Reply:
Ah OK, glad it was added later on at least.

Do we need to add it here as well though? This will add `importTime` for users _ahead_ of migration 119, which doesn't seem necessary.

in general I don't think we should use state types like this in migrations because migrations are meant to deal with state at a particular point in time. But types get updated over time. The `InternalAccount` type used in this migration doesn't reflect the shape of an internal account for a user running this migration.

Maybe we can remove the type from the migration instead, or use `// @ts-expect-error` to ignore the error, since it was caused by post-105 type changes

Reply:
It looks like adding `importTime` pre-emptively will not have any negative impact here because migration 119 will still replace it (it replaces any falsy `importTime` metadata, and 0 is falsy). But going forward it would be better to remove state types from migrations rather than updating them after the fact.